### ISSUE TITLE
Multiple spaces won't show up

### ISSUE THREAD

**Issue by Miepee** [Tyrrrz/DiscordChatExporter#121]
Multiple spaces won't show up in the html.
This is how Discord would show them:
```
-Title1:                
       *Data1               
       *Data2 
-Title2:
```

While the html shows it like this:
```
-Title1:
*Data1
*Data2
-Title2:

**Comment by Tyrrrz** [comment]
Can you give an expected/actual HTML code? Doesn't have to be 100% precise but just to illustrate the problem better.

**Comment by Miepee** [comment]
Sure.
Actual HTML Code:
```
-Title1:                                                             
       *Data1               
       *Data2 
-Title2:
```

Expected HTML Code:
```
-Title1:                                                             
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;*Data1               
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;*Data2 
-Title2:
```

This probably looks ugly, so using `&ensp;`  or `&emsp;` would probably be a better idea. (2 and 4 spaces respectively.)

**Comment by Tyrrrz** [comment]
Fixed in #145
